x-pack/plugins/security/index.js

```diff
@@ -38,6 +38,7 @@ export const security = (kibana) => new kibana.Plugin({
       encryptionKey: Joi.string(),
       sessionTimeout: Joi.number().allow(null).default(null),
       secureCookies: Joi.boolean().default(false),
+      loginAssistanceMessage: Joi.string(),
```

Comment:
__note:__ original PR used `.default()` here, but it's useless and does nothing.